File: Week 5/echon.pl
#!/usr/bin/perl -w



$rep= $ARGV[0];
$value= $ARGV[1];

$argLength= $#ARGV;


if ($argLength != 1){

	print STDERR "Usage: $0 <number of lines> <string>\n";

}

else {

	if ($rep =~ /^[0-9]/ ){
		
		for ($i=0 ; $i<$rep ; $i++){
		
			print "$value\n";

		}

	} 

	else {
		
		print STDERR "$0: argument 1 must be a non-negative integer\n";
	}

}

File: Week 7/echon.py
#!/usr/bin/env python3

import sys

def error():
    print ("Usage: {} <number of lines> <string>".format(sys.argv[0]))

if len(sys.argv) != 3:
    print (error())
    sys.exit()

else:
    print("blob")
    if sys.argv[1].isdigit() == False:
        print ("{}: argument 1 must be a non-negative integer".format(sys.argv[0]))
    else:
        for i in range(int(sys.argv[1])):
            print(sys.argv[2])

